siteapp/settings.py

```diff
@@ -136,6 +136,10 @@ def make_secret_key():
 	'simple_history.middleware.HistoryRequestMiddleware',
 	'pyinstrument.middleware.ProfilerMiddleware',
 ]
+CACHE_MIDDLEWARE_ALIAS='default'
+CACHE_MIDDLEWARE_SECONDS=604800
```

Comment:
@davidpofo Can you add some comments regarding what this cache does?

Reply:
I just removed as this was an attempt at site-wide caching that is more trouble than it is worth

Reply:
I have an idea on improving performance. Let's discuss after the rATO standup?